_posts/2018-09-13-september-office-hours.md

```diff
@@ -0,0 +1,87 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-13
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+  - kubernetes
+author_staff_member: burton
+dark_background: true
+---
+
+This month's OpenFaaS office hours meeting talks about OpenFaaS Cloud as a multi-user platform for teams, how we've improved the ability to run stateless microservices managed as functions, and includes a demo of Istio service mesh integration.
+
+The OpenFaaS community holds regular office hours for anyone to join in as well as more focused discussions with project members
+
+Types of calls:
+* Office Hours (750 community members)
+  * Questions and Answers
+  * Project updates and direction
+* Members Call (15 members)
+  * Members of the Github organization
+  * Assign and track issues
+* Core Contributors (5 core members)
+  * Wide project vision
+  * Help with project direction
+
+> All of the office hours meetings are recorded and available for later viewing on YouTube.
+
+<iframe width="700" height="394" src="https://www.youtube.com/embed/IQCnHMn1wdI?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+Subscribe to the [Official OpenFaaS YouTube Channel](https://www.youtube.com/channel/UCdKi97g5FmzvrmtIp9FyOVA) where you can view the highlights from our meetings including demos of the newest features we're working on
+
+## Project Goals and Purpose
+OpenFaaS is focused on developers first, easy for operators, and the great community aspect
+
+* Developer first
+  * OpenFaaS gives you "Serverless on your terms"
+  * Focused on being easy to use with a web UI and CLI (eg: `faas-cli up`)
+    * [Five Tips and Tricks for the OpenFaaS CLI]({% post_url 2018-08-22-five-cli-tips %})
+* Easy for operators
+  * [OpenFaaS Operator](https://github.com/openfaas-incubator/openfaas-operator)
+  * [Scale to Zero]({% post_url 2018-07-25-zero-scale %})
+* Great Community
+  * 130+ contributors
+  * Most have multiple commits, coming back helping build the project
+  * Built by the community, for you to use
```

Comment:
This feels out of place with the post description - warrants a seperate blog post about project goals perhaps.

I'd potentially move all of the sections about the community meetings to the. And I'd group them under a header.

The title/description of the post is about describing what was demoed/discussed, not describing that a community meeting is, hence pushing that to the end as a secondary section. (It potentially warrants being a post of its own perhaps - a broader one on how a community driven project organises and disseminates information/feedback etc)

Reply:
This was included as it was a speaking point in the talk. It's not a main point of the meeting, but was brought up. As this is a recap of the meeting, I thought it would be appropriate to include.

I do agree that we should write something up highlighting the various meetings, their purpose, and how they are successful.